village/src/App.js

```diff
@@ -1,8 +1,13 @@
-import React, { Component } from 'react';
 
+import React, { Component } from 'react';
+import { Route, NavLink } from 'react-router-dom';
+import axios from 'axios';
```

Comment:
Good addition here Scott; this is what was missing in the last iteration, and you found the solution. Well done!